ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -225,6 +214,12 @@ const AdminUser = withUrlParams()(class extends React.Component<Props, State> {
     }
   }
 
+  getFreeCreditLimitOptions() {
+    const {oldProfile: {freeTierDollarQuota}} = this.state;
+    const defaultsPlusMaybeOverride = Array.from(freeCreditLimitDefaultOptions.add(freeTierDollarQuota));
```

Comment:
tested with quota:
* undefined
* equal to a predefined value: does not duplicate
* defined, with various values in and around the predefined range: orders correctly

Reply:
I'm not sure if we want to tolerate undefined values or just assume we'll set whatever isn't there to `null`. IOW I never understood how `undefined` is a feature of JavaScript.

Reply:
Here I was imprecise: I do not mean JS `undefined`.  Rather, the DB user has `free_tier_credits_limit_dollars_override` set to NULL, which is the state for almost all users.

This results in `freeTierDollarQuota` here being equal to the config-file default value, which may or may not be one of the predefined dropdown options (300 to 800) but in current practice is 300.